compiler/commands.nim

```diff
@@ -34,9 +34,6 @@ from ast import eqTypeFlags, tfGcSafe, tfNoSideEffect
 
 # but some have deps to imported modules. Yay.
 bootSwitch(usedTinyC, hasTinyCBackend, "-d:tinyc")
```

Comment:
in future work, all this can probably be replaced with `compilesettings(commandLine)` or similar